### PR TITLE
Fix lack of optimisation for SHAPEIT4 v4.1.3

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Fixes issue #12204

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/e0e6ab91bfbffdae662f355e297b80ac for a full test report.

**Review by boegel — CHANGES_REQUESTED** [review]
@verdurin I think we can avoid a patch file here by just using `buildopts`, see also #12207

**Comment by verdurin** [comment]
@boegel I think we still need to patch out hard-coded `HTSlib` versions etc.

File: easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4-4.1.3-foss-2019b.eb [hunk 1]
```diff
@@ -31,6 +31,7 @@ dependencies = [
 
 buildopts = 'HTSLIB_INC="$EBROOTHTSLIB/include" HTSLIB_LIB=-lhts '
 buildopts += 'BOOST_INC="$EBROOTBOOST/include" BOOST_LIB_IO=-lboost_iostreams BOOST_LIB_PO=-lboost_program_options'
```
Comment (boegel) [inline easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4-4.1.3-foss-2019b.eb:33]: @verdurin Needs a space at the end here...

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/cde9356413305ee657c78714e297d584 for a full test report.

File: easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4-4.1.3-foss-2019b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/SHAPEIT4/SHAPEIT4-4.1.3-foss-2019b.eb]: fixed in 75d223d

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3101.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/4d2491d1ea46f8979c7b6c762508d4cc for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**